sops/alerts/container_waiting.adoc

```diff
@@ -0,0 +1,20 @@
+:toc:
+:toc-placement!:
+
+= ThreescaleContainerWaiting
+
+toc::[]
+
+== Description
+
+* This alert will trigger if a container has been in `waiting` state for longer than 1 hour
```

Comment:
Pods that cannot be scheduled may be also here? For example, due to lack of resources.